src/plugin/featureaction/featureactionmanager.js

```diff
@@ -309,9 +309,15 @@ plugin.im.action.feature.Manager.prototype.addSource_ = function(source) {
       this.sourceListeners_[id] = ol.events.listen(/** @type {ol.events.EventTarget} */ (source),
           goog.events.EventType.PROPERTYCHANGE, this.onSourcePropertyChange_, this);
 
-      this.loadDefaults(id).thenAlways(function() {
+      var promise = os.layer.preset.LayerPresetManager.getInstance().getPresets(id, true);
```

Comment:
I'm not sure I'm following this change... when is the `loadDefaults` call happening now? Is the preset manager now completely responsible for that?

Reply:
Yeah, that "true" param makes it so when a new source is added, it'll look for and apply whatever "default=true" preset it detects - if it hasn't done so before (i.e. it'll rely on the User's current settings if it the layer/source has been added before).

Reply:
The reason for my trepidation here is that the original intent of default feature actions in this manager was for them to be an independent piece of functionality that could hypothetically be loaded from anywhere (not just a preset)... Not sure if such a thing could still be supported just by looking at this PR.

Reply:
ImportActionManager.loadDefaults(id) is called by SettingsPresetService through...
`lpm.getPresets(id)` >> `for presetservices...` >> `sps.find(id)` >> `iam.loadDefaults(id)`
There aren't settings.json configs such that the sps.find() wouldn't end up calling loadDefaults(). It's incredibly circuitous compared to "this.loadDefaults()"; but opens the door for Default FA's that aren't deployed with the code and/or settings.json like how loadDefaults() requires.